### DISCUSSION TITLE
How to connect hudi with Aliyun OSS?

### DISCUSSION THREAD

**Discussion by songpcmusic** [trinodb/trino#18710] [Q&A]
`hudi.properties` can connect with S3/MinIO, how to connect hudi with Aliyun OSS, is there any solutions? Thanks.

**Comment by mxdzs0612** [comment]
Both S3SDK and JindoSDK are available.

**Reply by gfunc** [reply]
any approach to support the scheme (oss://) ? since the connected hive has oss://xxx for file locations.

**Reply by mxdzs0612** [reply]
Configs in core-site file are needed. See https://github.com/aliyun/alibabacloud-jindodata/blob/master/docs/user/4.x/4.6.x/4.6.11/oss/presto/jindosdk_on_presto.md if you can read Chinese.

**Comment by electrum** [comment]
What is required for this? Can you simply change the S3 endpoint?

**Reply by mxdzs0612** [reply]
OSS implementation with `oss://` prefix is needed using S3 SDK.
